### DISCUSSION TITLE
starting new project with react

### DISCUSSION THREAD

**Discussion by zeev-mindali** [zwave-js/zwave-js#6618] [Q&A] [answered]
Dear all,

i want to start my own project in react-typescript.
made a install for zwave-js
and imported the drive.
once i starting the program i getting over 75 error, and a web pack issue

using react 18.2.0;

what am i missing?
i want for start to get all the nodes in my zwave networkds and thier epids

**Comment by AlCalzone** [comment, marked as answer]
Z-Wave JS is meant to run on a Node.js backend. You cannot just use it in React. Only the stuff imported from the ".../safe" entrypoints can be used directy on the frontend.
What you could do is run https://github.com/zwave-js/zwave-js-server on the backend and the communicate with that using websockets from React.
